.github/workflows/trigger-load-tests.yml

```diff
@@ -0,0 +1,52 @@
+name: Trigger load tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      tests:
+        description: >
+          List of test names. This needs to be filled only if you want to run a specific set of tests. Example: foo,bar
+        required: false
+      clusterName:
+        description: 'Cluster name'
+        default: 'io-perf-cluster-test'
+        required: false
+
+jobs:
+  invoke-load-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Get latest ballerina runtime artifact
+        id: setRuntimeUrl
+        shell: bash
+        run: |
+          printf "::set-output name=runtimeUrl::%s" $(curl -H "Accept: application/vnd.github.v3+json" \
+            https://api.github.com/repos/ballerina-platform/module-ballerina-http/actions/artifacts | \
+            jq '[.artifacts[] | select(.name=="ballerina-runtime" and .expired==false)]|max|.archive_download_url')
+      - name: Format test names
+        id: formatTestNames
+        shell: bash
+        run: |
+          tests="${{github.event.inputs.tests}}"
+          if [ ! "$tests" = "" ]; then
+            testsInput=(${tests//,/ })
+            testsCount=${#testsInput[@]}
+            testsOutput=""
+            for (( i=0; i<$testsCount; i++ ));
+            do
+                testsOutput+=\\'"'${testsInput[$i]}\\'"',
+            done
+            printf "::set-output name=testNames::%s" [${testsOutput:0:-1}]
+          else
+            printf "::set-output name=testNames::%s" $tests
+          fi
+      - name: Invoke standard library load test workflow
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: Stdlib Workflow
+          repo: ballerina-platform/ballerina-performance-cloud
+          ref: refs/heads/main
+          token: ${{ secrets.BALLERINA_BOT_TOKEN }}
+          inputs: '{ "repo-name": "module-ballerina-io", "tests": "${{ steps.formatTestNames.outputs.testNames }}", "zipURL": ${{ steps.setRuntimeUrl.outputs.runtimeUrl}}, "clusterName": "${{ github.event.inputs.clusterName }}" }'
```

Comment:
Need to update here also. https://github.com/ballerina-platform/module-ballerina-grpc/blob/master/.github/workflows/trigger-load-tests.yml

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-io/pull/416/commits/0ba2e1ba77dfe2f668930abefaa0aeea7ddc024f